go/Makefile

```diff
@@ -8,33 +8,30 @@ deps:
 	@go install google.golang.org/protobuf/cmd/protoc-gen-go@latest;
 	@go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@latest;
 
-protoc-build: deps
-ifneq ($(shell git diff --quiet HEAD ${REF} -- ../proto || echo "changed"),)
-	$(call outmsg, $(MODULE), "API Change detected")
-else
-	@$(MAKE) proto-gen
-endif
-
-build: protoc-build
+build: deps proto-gen
 
 proto-gen:
 	$(call outmsg, $(MODULE), "Generating golang proto bindings")
 	@./proto-gen.sh
 
 test:
 	$(call outmsg, $(MODULE))
-	@go test  -failfast -short ./pkg/...
+	@go test  ./... -failfast -short --coverprofile=coverage.txt
 
 clean:
 	$(call outmsg, $(MODULE))
 	@go clean
+ifneq ($(wildcard api/.*),)
+	@find ./receptor_v1 -name "*.pb.*" -type f -delete
+	@find ./receptor_v1 -type d -empty -delete
+endif
 
 fmt:
 	$(call outmsg, $(MODULE))
-	@go fmt -x ./...
+	@go fmt -x ./... > /dev/null
```

Comment:
Why dev null?

Reply:
it prints to stdout the list of go files it will format. Not so interesting and may be confusing when doing a `make all`:


```
🧹 Cleaning build files ........................................................................................................................................................ [trustero-api-go]
⬇️  Fetching dependencies ....................................................................................................................................................... [trustero-api-go]
🦄 Generating golang proto bindings ............................................................................................................................................ [trustero-api-go]
✍️  Applying code style ......................................................................................................................................................... [trustero-api-go]
/opt/homebrew/opt/go/libexec/bin/gofmt -l -w examples/directory-receptor/directory_receptor.go examples/directory-receptor/main.go examples/directory-receptor/generators/credentials.go examples/directory-receptor/generators/directory.go examples/directory-receptor/generators/list_all.go receptor_sdk/client/client.go receptor_sdk/cmd/receptor.go receptor_sdk/cmd/root.go receptor_sdk/cmd/scan.go receptor_sdk/cmd/scan_test.go receptor_sdk/cmd/utils.go receptor_sdk/cmd/verify.go receptor_sdk/config/cert.go receptor_sdk/config/log.go receptor_sdk/utils/json.go receptor_v1/document.pb.go receptor_v1/evidence.pb.go receptor_v1/finding.pb.go receptor_v1/receptor.go receptor_v1/receptor.pb.go receptor_v1/receptor_grpc.pb.go receptor_v1/struct.pb.go
🧹 Cleaning build files ............................................................................................................................................................ [api-gen-doc]
⬇️  Fetching dependencies ........................................................................................................................................................... [api-gen-doc]
🦄 Generate API reference documentation ............................................................................................................................................ [api-gen-doc]
⚙️  Build done ...................................................................................................................................................................... [api-gen-doc]
🚧 Nothing to be done for 'fmt' .................................................................................................................................................... [api-gen-doc]
```